### ISSUE TITLE
"uid 30001 is greater than SYS_UID_MAX 999" when installing on Fedora 37

### ISSUE THREAD

**Issue by bjorn** [nix-community/nix-installers#16]
It appears to be just a warning, but I couldn't find it already reported, so I just wanted to make sure people are aware. I don't recall any configuration changes I might have made to trigger this warning.

Full installation log:
```
$ sudo dnf localinstall nix-multi-user-2.9.1.rpm 
Dependencies resolved.
================================================================================================================================
 Package                           Architecture              Version                      Repository                       Size
================================================================================================================================
Installing:
 nix-multi-user                    x86_64                    2.9.1-1                      @commandline                     40 M

Transaction Summary
================================================================================================================================
Install  1 Package

Total size: 40 M
Installed size: 40 M
Is this ok [y/N]: y
Downloading Packages:
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                        1/1 
  Installing       : nix-multi-user-2.9.1-1.x86_64                                                                          1/1 
  Running scriptlet: nix-multi-user-2.9.1-1.x86_64                                                                          1/1 
useradd warning: nixbld1's uid 30001 is greater than SYS_UID_MAX 999
useradd warning: nixbld2's uid 30002 is greater than SYS_UID_MAX 999
useradd warning: nixbld3's uid 30003 is greater than SYS_UID_MAX 999
useradd warning: nixbld4's uid 30004 is greater than SYS_UID_MAX 999
useradd warning: nixbld5's uid 30005 is greater than SYS_UID_MAX 999
useradd warning: nixbld6's uid 30006 is greater than SYS_UID_MAX 999
useradd warning: nixbld7's uid 30007 is greater than SYS_UID_MAX 999
useradd warning: nixbld8's uid 30008 is greater than SYS_UID_MAX 999
useradd warning: nixbld9's uid 30009 is greater than SYS_UID_MAX 999
useradd warning: nixbld10's uid 30010 is greater than SYS_UID_MAX 999
useradd warning: nixbld11's uid 30011 is greater than SYS_UID_MAX 999
useradd warning: nixbld12's uid 30012 is greater than SYS_UID_MAX 999
useradd warning: nixbld13's uid 30013 is greater than SYS_UID_MAX 999
useradd warning: nixbld14's uid 30014 is greater than SYS_UID_MAX 999
useradd warning: nixbld15's uid 30015 is greater than SYS_UID_MAX 999
useradd warning: nixbld16's uid 30016 is greater than SYS_UID_MAX 999
useradd warning: nixbld17's uid 30017 is greater than SYS_UID_MAX 999
useradd warning: nixbld18's uid 30018 is greater than SYS_UID_MAX 999
useradd warning: nixbld19's uid 30019 is greater than SYS_UID_MAX 999
useradd warning: nixbld20's uid 30020 is greater than SYS_UID_MAX 999
useradd warning: nixbld21's uid 30021 is greater than SYS_UID_MAX 999
useradd warning: nixbld22's uid 30022 is greater than SYS_UID_MAX 999
useradd warning: nixbld23's uid 30023 is greater than SYS_UID_MAX 999
useradd warning: nixbld24's uid 30024 is greater than SYS_UID_MAX 999
useradd warning: nixbld25's uid 30025 is greater than SYS_UID_MAX 999
useradd warning: nixbld26's uid 30026 is greater than SYS_UID_MAX 999
useradd warning: nixbld27's uid 30027 is greater than SYS_UID_MAX 999
useradd warning: nixbld28's uid 30028 is greater than SYS_UID_MAX 999
useradd warning: nixbld29's uid 30029 is greater than SYS_UID_MAX 999
useradd warning: nixbld30's uid 30030 is greater than SYS_UID_MAX 999
useradd warning: nixbld31's uid 30031 is greater than SYS_UID_MAX 999
useradd warning: nixbld32's uid 30032 is greater than SYS_UID_MAX 999

  Verifying        : nix-multi-user-2.9.1-1.x86_64                                                                          1/1 

Installed:
  nix-multi-user-2.9.1-1.x86_64                                                                                                 

Complete!
```

**Comment by yajo** [comment]
Isn't [nix 2.12 now able to work in multi-user mode without nixbld users](https://discourse.nixos.org/t/nix-2-12-0-released/23780)?

If that's the case, that should make Nix work better out of the box in other distros, if updated in the package. That should make installation in rpm-ostree distros also much easier (see #8).

**Comment by yajo** [comment]
> For me install on Fedora 37 breaks the system, after receiving the same warnings. Next boot results in a flickering screen. Tried the install 3 times, after rolling back with snapper, always the same issue.

That's another problem, fixed in #19 already.
